### ISSUE TITLE
Evaluation of arxiv

### ISSUE THREAD

**Issue by KennyNH** [snap-stanford/prodigy#3]
In the evaluation of arxiv, why the param "ignore_label_embeddings" should be set as True and the label embeddings are generated by the nn.Embedding layer? Why don't you use the label embeddings generated by bert?

**Comment by q-hwang** [comment]
I I remember correctly, we found that there is not much difference between the two options. So we chose to keep the setting consistent with pretraining.
